### ISSUE TITLE
Fix local startup

### ISSUE THREAD

**Issue by saracope** [GSA/code-gov-front-end#329]
**User story:**
`As a contributor, I would like clear instructions that properly describe how to run the code.gov site locally on my machine.`  

**Issue:**
Running `npm run start` locally after install doesn't start the application with any projects or tasks (see screenhots). The [README.md](README.md) indicates that if API info isn't provided, the system will use the defaults from the [site.json](https://github.com/GSA/code-gov-front-end/blob/master/config/site/site.json) file. That doesn't seem to be actually happening. This has proved confusing to users in testing interviews.  

**Describe the bug**

Site isn't getting the API info at startup by default.
Partial output at startup:
```
process.env.CODE_GOV_API_BASE: undefined
process.env.CODE_GOV_API_KEY: undefined
process.env.CODE_GOV_BRANCH: undefined
process.env.CODE_GOV_RELATIVE_DIR: undefined
process.env.PUBLIC_PATH: undefined
OUTPUT_PATH: /Users/sarassassin/Documents/Websites/code-gov-repos/code-gov-front-end/dist
process.env.CODE_GOV_CONFIG_JSON: undefined
```
**To Reproduce**

1. Clone repo
2.  Run `npm install`
3. Run `npm run start`
4. Look at Browse Projects and Open Tasks pages on local server to see that no projects are listed on either page

**Expected behavior**

A contributor should be able to run the site locally on their machine without needing additional help or instructions from a code.gov engineer or 3rd party.

This Issue is completed when:
[ ] Running `npm run start` will run the site locally using the default Project and Task data provided in the site.json file
[ ] The Open Tasks API info is included in the site.json file
[ ] The code.gov documentation is updated to clearly reflect the steps needed to run the site locally on the users machine

**Screenshots**

<img width="1312" alt="Screen Shot 2019-11-20 at 9 46 37 AM" src="https://user-images.githubusercontent.com/2197515/69249203-8183c180-0b7b-11ea-8ed3-3db529e6d2ac.png">
<img width="1413" alt="Screen Shot 2019-11-20 at 9 46 28 AM" src="https://user-images.githubusercontent.com/2197515/69249205-8183c180-0b7b-11ea-8d88-8f9198848160.png">

**Comment by afomi** [comment]
@saracope - i was able to get this repo up and running in the last few days. Happy to chat about specifics to address this issue.

I did need to get a key from https://developers.code.gov/key.html

**Comment by jcastle** [comment]
This is working better now with changes made to .env file as addressed with PR #368 & #370
